draft-ietf-quic-recovery.md

```diff
@@ -519,6 +519,26 @@ bytes.
 Initial packets and Handshake packets may never be acknowledged, but they are
 removed from bytes in flight when the Initial and Handshake keys are discarded.
 
+### Speeding Handshake Completion
+
+When a server receives duplicate Initial CRYPTO data, it can assume the client
+did not receive all Initial CRYPTO data or the client's estimated RTT is too
+small. When a client receives Handshake packets prior to obtaining Handshake
+keys it may assume some or all of the server's Initial packets were lost.
```

Comment:
As there's a chance where a client receives an 1-RTT packet before it receives Initial and Handshake packets, I think it'd be more natural to cover that case too.

Maybe s/When a client receives Handshake packets/When a client receives Handshake or 1-RTT packets/.